src/main/kotlin/me/spradling/gift/core/conversion/GiftCommitApiConverter.kt

```diff
@@ -0,0 +1,22 @@
+package me.spradling.gift.core.conversion
+
+import io.vertx.core.Future
+import me.spradling.gift.core.api.models.Account
+import java.util.stream.Collectors
+
+class GiftCommitApiConverter {
+
+  fun convert(accounts: Future<List<me.spradling.gift.core.database.models.Account>>): List<Account> {
+
+    return if (accounts.result() != null) {
```

Comment:
This isn't going to work. You should avoid using `.result()` if you can because it is only safe unless you are absolutely sure the `Future` has been completed. Which in this case, it might not be. 

The more appropriate way to do this would be in the handler composing the future and calling the convert with the actual result. This method should take in a List<Account> instead of a Future.